packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -79,12 +79,82 @@ const getInfo = async (request: Request<MessageTypes.GetInfo>) => {
     } as const;
 };
 
+const subscribeBlock = async ({ state, connect, post }: Context) => {
```

Comment:
I included this subscribe method in this PR, even tho it may be outside of `minimal implementation` because I wanted to showcase an endpoint which uses websocket. It turned out a bit different in the end so a showcase of the WebSocket will be implemented in future PR (`onAccountChange`)